lib/tasks/define_permissions.rake

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+# RAILS_ENV=production bundle exec rake permissions:dc
+# RAILS_ENV=production bundle exec rake permissions:me
+# RAILS_ENV=production bundle exec rake permissions:set_role_and_permissions email="admin@ideacrew.com" role='super_admin'
+
+namespace :permissions do
```

Comment:
good question. 

this is a rake which gets triggered manually and won't be executed by server. And, we're just creating a bunch of permissions which we will only do once - so, i didn't see much value in adding one.

Reply:
Ok, thanks for the explanation!  Like I said, I have not come across any issues testing locally.  I think that should suffice considering the two points you made-- this will be run manually and only one time.